### PR TITLE
Add support for the Multz option ALL in PINCH

### PR DESCRIPTION
**Pull request by totto82**
This needs some more testing 

- [x] what happens when different means of multiplying the transmissibility in the z direction is considered. 
- [x] test for MULTZ-

### CODE REVIEW

File: ebos/ecltransmissibility.hh [hunk 1]
```diff
@@ -323,7 +324,32 @@ public:
 
                 // apply the full face transmissibility multipliers
                 // for the inside ...
-                applyMultipliers_(trans, insideFaceIdx, insideCartElemIdx, transMult);
+
+                // The MULTZ needs special case if the option is ALL
```
Comment (andlaus) [inline ebos/ecltransmissibility.hh]: could the new code moved to `applyMultipliers_()`

Comment (andlaus) [reply ebos/ecltransmissibility.hh]: yes, that would be better. this method is quite a bit too long as it is, IMO.

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
This is ready from my side. @andlaus Can this be merged?

**Comment by alfbr** [comment]
This has been tested and confirmed to solve the problem on our side.

**Comment by andlaus** [comment]
looks good. thanks for the patch.

**Pull request merged**